pkgs/development/python-modules/pylatex/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, matplotlib
+, numpy
+, ordered-set
+, quantities
+, withMatplotlib ? true
+, withMatrices ? true
+, withQuantities ? true
+}:
+
+buildPythonPackage rec {
+  pname = "PyLaTeX";
+  version = "1.3.4";
```

We should stick to supported Python releases for new packages..

```suggestion
  version = "1.3.4";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```